modules/cdp_engine/benchmarking/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
either change this to `#[cfg(any(feature = "runtime-benchmarks", test))]` or change the cfg in mock.rs